recipes/geodemora/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "geodemora" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 9e8eb116e37d86690702a219a3283af3b09cc166647b0134cf04310c54de9e31
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
```

```suggestion
    - python >=3.7
```
Ref: https://github.com/ramendra1990/geodemora/blob/2cee3b7398f95baa4433c03573f5b58879ef1d58/setup.py#L30